ultravox/data/text_proc.py

```diff
@@ -31,15 +40,17 @@ def format_asr_text(text: str) -> str:
     remaining_words = []
     for word in text.split():
         if word in GIGASPEECH_GARBAGE_UTTERANCE_TAGS:
-            return ""
+            raise GarbageUtteranceError(f"Garbage utterance tag found: {word}")
```

Comment:
This will remove the utterance all together. I think it is better than "", but we may want to revisit this and include these utterances as they may help improve the robustness of the model.

Reply:
I don't see this error being handled in `datasets.py`, that means this is going to break training of any dataset that uses this function. Same for the other error.

Reply:
I think dropping the samples in `dataset.py` is good enough for now.